Ports/cmake/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=cmake
+version=3.19.4
+useconfigure=false
+files="https://github.com/Kitware/CMake/releases/download/v$version/cmake-$version.tar.gz cmake-$version.tar.gz"
+depends=""
+
+port_path=$(realpath $(dirname ${BASH_SOURCE[0]}))
+
+build() {
+    return
+}
+
+install() {
+    cmake_dir="${SERENITY_ROOT}/Build/Root/home/anon/Source/cmake"
+    rm -rf "$cmake_dir"
+    mkdir -p "$cmake_dir"
```

Comment:
I don't see harm in using `run` here as well, it `cd`'s needlessly but at least that makes it print the current command to see what's going on.

```suggestion
    run rm -rf "$cmake_dir"
    run mkdir -p "$cmake_dir"
```

Reply:
Sure, why not :shrug: